costmap_2d/src/layered_costmap.cpp

```diff
@@ -260,6 +260,17 @@ bool LayeredCostmap::isCurrent()
   return current_;
 }
 
+bool LayeredCostmap::areAllLayersInitialized() {
+  bool initialized = true;
+   for (vector<boost::shared_ptr<CostmapLayer> >::iterator plugin = plugins_.begin(); plugin != plugins_.end();
```

Comment:
if we don't have plugins setup properly (in very extreme bad case), is it correct to say we are initialized? should default be false instead of true?